pkg/pillar/cmd/zedrouter/zedrouter.go

```diff
@@ -2870,3 +2877,47 @@ func releaseAppNetworkResources(ctx *zedrouterContext, key string,
 	}
 	publishAppNetworkStatus(ctx, status)
 }
+
+func isDNSServerChanged(new *types.DeviceNetworkStatus) bool {
+	for _, port := range new.Ports {
+		if port.IsMgmt {
```

Comment:
We can have a local network instance on a non-management port, and in that case we also need to do the restart. Thus I think you can remove the IsMgmt check.

Reply:
sure, will remove.